docs/developer-guide/debugging.md

```diff
@@ -0,0 +1,42 @@
+# Debugging using VSCode and lldb on a remote machine or VM 
+
+1. Install [code-lldb vscode extension](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)
+2. Add a configuration to `.vscode/launch.json` like the following:
+    ```json
+    {
+        "name": "Remote debug bpfd",
+        "type": "lldb",
+        "request": "launch",
+        "program": "<ABSOLUTE_PATH>/github.com/bpfd-dev/bpfd/target/debug/bpfd", // Local path to latest debug binary.
+        "initCommands": [
+            "platform select remote-linux", // Execute `platform list` for a list of available remote platform plugins.
+            "platform connect connect://<IP_ADDRESS_OF_VM>:8081", // replace <IP_ADDRESS_OF_VM>
+            "settings set target.inherit-env false",
+        ],
+        "env": {
+            "RUST_LOG": "debug"
+        },
+        "cargo": {
+            "args": [
+                "build",
+                "--bin=bpfd",
+                "--package=bpfd"
+            ],
+            "filter": {
+                "name": "bpfd",
+                "kind": "bin"
+            }
+        },
+        "cwd": "${workspaceFolder}",
+    },
+    ``` 
+3. On the VM or Server install `lldb-server`
+    ```bash
+        sudo apt install lldb-server
+    ```
```

Comment:
```suggestion
    `dnf` based OS:

    ```bash
        sudo dnf install lldb-server
    ```

    `apt` based OS:

    ```bash
        sudo apt install lldb-server
    ```
```

However, on Fedora:
```
$ sudo dnf install lldb-server
Last metadata expiration check: 3:03:50 ago on Wed 04 Oct 2023 04:08:54 AM EDT.
No match for argument: lldb-server
Error: Unable to find a match: lldb-server
```
So what is the equivalent, just `sudo dnf instal lldb`?

Reply:
Yeah it's actually included in lldb I believe